calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/modules/OaiHarvestConfigModule.scala

```diff
@@ -5,7 +5,7 @@ import javax.inject.Singleton
 import com.google.inject.Provides
 import com.twitter.inject.TwitterModule
 
-case class OaiHarvestConfig(oaiUrl: String)
+case class OaiHarvestConfig(oaiUrl: String, oaiDaysToFetch: Long)
```

Comment:
Just the `OaiHarvestConfig` model. In this case it doesn't need to be in common though:

Probably:

`calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/models/`